tpm2/tpm2_test.go

```diff
@@ -16,21 +16,54 @@ package tpm2
 
 import (
 	"bytes"
+	"context"
 	"crypto"
 	"crypto/ecdsa"
 	"crypto/elliptic"
 	"crypto/rand"
 	"crypto/rsa"
 	"crypto/sha256"
 	"flag"
+	"io"
 	"math/big"
 	"os"
 	"reflect"
 	"testing"
 
 	"github.com/google/go-tpm/tpmutil"
+	"github.com/google/go-tpm/tpmutil/mssim"
 )
 
+// To run integration tests against the Microsoft simulator default ports, run:
+// go test github.com/google/go-tpm/tpm2 -args --mssim_command_addr=localhost:2321 --mssim_platform_addr=localhost:2322
```

Comment:
Set these defaults on flag definitions below

Reply:
added the defaults, but had to re-add a boolean flag so the user has something to indicate they want to run the tests against a simulator